<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTTESTClass</class>
 <widget class="QMainWindow" name="QTTESTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTTEST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="pitchEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="beepButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Beep [pitch] [length]:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_trayButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Tray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CMD:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cmdEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Payload</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="generateEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCommand_List"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCommand_List">
   <property name="text">
    <string>Command List</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qttest.qrc"/>
 </resources>
 <connections/>
</ui>
